Examples/Swift/ShapeCollectionFeatureExample.swift

```diff
@@ -0,0 +1,70 @@
+import Mapbox;
+
+let MapboxAccessToken = "<# Access Token #>"
```

Comment:
Sorry, but now that I see this external variable up here I’m thinking that it would be better to just include `<# Mapbox Access Token #>` in the URL string itself. In our Swift library examples the external variable exists so that we might throw an exception when the user hasn’t set it, which doesn’t need to be the case here.